### ISSUE TITLE
Processor NOT matched Haswell i7-4700MQ

### ISSUE THREAD

**Issue by jonathansolorzn** [Piker-Alpha/ssdtPRGen.sh#175]
I've followed the steps, I also checked that it's a supported processor, but I'm getting the following message:

```
Jonathans-MacBook-Pro:~ feniixx$ ./ssdtPRGen.sh -p 'i7-4700MQ' -f 2400 -turbo 3400


ssdtPRGen.sh v0.9 Copyright (c) 2011-2012 by † RevoGirl
             v6.6 Copyright (c) 2013 by † Jeroen
             v16.4 Copyright (c) 2013-2015 by Pike R. Alpha
-----------------------------------------------------------
Bugs > https://github.com/Piker-Alpha/ssdtPRGen.sh/issues <

./ssdtPRGen.sh: line 3406: /Users/feniixx/Library/ssdtPRGen/Data/Models.cfg: No such file or directory
Override value: (-p) processor model, now using: i7-4700MQ!
Override value: (-f) clock frequency, now using: 2400 MHz!
Override value: (-turbo) maximum (turbo) frequency, now using: 3400 MHz!

System information: Mac OS X 10.11.2 (15C50)
Brandstring 'Intel(R) Core(TM) i7-4700MQ CPU @ 2.40GHz'

gModelDataVersion: 
./ssdtPRGen.sh: line 2707: /Users/feniixx/Library/ssdtPRGen/Data/Ivy Bridge.cfg: No such file or directory
./ssdtPRGen.sh: line 2718: /Users/feniixx/Library/ssdtPRGen/Data/Haswell.cfg: No such file or directory
./ssdtPRGen.sh: line 2729: /Users/feniixx/Library/ssdtPRGen/Data/Broadwell.cfg: No such file or directory
./ssdtPRGen.sh: line 2740: /Users/feniixx/Library/ssdtPRGen/Data/Skylake.cfg: No such file or directory
Scope (_PR_) {1644 bytes} with ACPI Processor declarations found in the DSDT (ACPI 1.0 compliant)
Generating ssdt.dsl for a 'MacBookPro11,1' with board-id [Mac-189A3D4F975D5FFC]
Haswell Core i7-4700MQ processor [0x306C3] setup [0x0705]
Processor NOT matched, checking required arguments!

Error: Low Frequency Mode is 0 ...
Aborting ...
Done.

```

Has it something to do with "Macbook 11,1"? What am I supposed to do?

**Comment by CYBOPOB** [comment]
I had got same result
ssdtPRGen.sh v0.9 Copyright (c) 2011-2012 by † RevoGirl
             v6.6 Copyright (c) 2013 by † Jeroen

##              v16.5 Copyright (c) 2013-2015 by Pike R. Alpha

Bugs > https://github.com/Piker-Alpha/ssdtPRGen.sh/issues <

/Users/SUVOROV/ssdtPRGen.sh: line 3406: /Users/SUVOROV/Library/ssdtPRGen/Data/Models.cfg: No such file or directory
System information: Mac OS X 10.11.2 (15C50)
Brandstring 'Intel(R) Core(TM) i7-3520M CPU @ 2.90GHz'

gModelDataVersion: 
/Users/SUVOROV/ssdtPRGen.sh: line 2707: /Users/SUVOROV/Library/ssdtPRGen/Data/Ivy Bridge.cfg: No such file or directory
/Users/SUVOROV/ssdtPRGen.sh: line 2718: /Users/SUVOROV/Library/ssdtPRGen/Data/Haswell.cfg: No such file or directory
/Users/SUVOROV/ssdtPRGen.sh: line 2729: /Users/SUVOROV/Library/ssdtPRGen/Data/Broadwell.cfg: No such file or directory
/Users/SUVOROV/ssdtPRGen.sh: line 2740: /Users/SUVOROV/Library/ssdtPRGen/Data/Skylake.cfg: No such file or directory
Scope (_PR_) {220 bytes} with ACPI Processor declarations found in the DSDT (ACPI 1.0 compliant)
Generating ssdt.dsl for a 'MacBookAir5,2' with board-id [Mac-2E6FAB96566FE58C]
Ivy Bridge Core i7-3520M processor [0x306A9] setup [0x0704]
Processor NOT matched, checking required arguments!

Error: Low Frequency Mode is 0 ...
Aborting ...
Done.

**Comment by Piker-Alpha** [comment]
This issue should now be fixed (see issue #181).
